nebula/ui/nebula_resources.qrc

```diff
@@ -307,7 +307,6 @@
         <file>resources/Icon1024.png</file>
         <file>resources/location-dark.svg</file>
         <file>resources/logo-dock.png</file>
-        <file>resources/logo.ico</file>
```

Comment:
How is this change related?

Reply:
this icon is totally not used. It's a dup of what we have in apps/vpn/ui/resources/logo.ico.